### PR TITLE
Fixes #3809 - Restart-Computer and Get-Content for style, formatting and clarity

### LINKED ISSUE #3809: Example 5 Misleading
This code snippet for example 5 is wrong. The snippet I am talking about:
"The first command uses the Get-Content cmdlet to get a list of computers in the domain from the Domain01.txt file. It stores the list in the $s variable.
PS C:\&gt; $s = Get-Content Domain01.txt"

All that does is restart all the computers whose name is in the text file. No where does this example grab all the computers in the domain and put them in the text file.

Suggested Correction:
The title should be, "Example 5: Force restart of all computers in a text file."

The code snippet should say, "The first command uses the Get-Content cmdlet to get a list of the computers in the Domain01.txt file. It stores the list in the $s variable."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5b56646b-9f1d-3f94-64b7-1e556c9ad803
* Version Independent ID: c4358932-1228-6e20-37e3-5ded6a469b8a
* Content: [Restart-Computer (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/restart-computer?view=powershell-6)
* Content Source: [reference/6/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Management/Restart-Computer.md)
* Product: **powershell**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[a07e12a](https://github.com/bobbytreed/PowerShell-Docs/pull/3840/commits/a07e12ad61bc6bccee92926fce32fc1671e24373)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/3.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-3.0&branch=pr-en-us-3840) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/3.0/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-3.0&branch=pr-en-us-3840) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/4.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-4.0&branch=pr-en-us-3840) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/4.0/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-4.0&branch=pr-en-us-3840) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/5.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.0&branch=pr-en-us-3840) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/5.0/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-5.0&branch=pr-en-us-3840) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/5.1/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.1&branch=pr-en-us-3840) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/5.1/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-5.1&branch=pr-en-us-3840) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/6/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-6&branch=pr-en-us-3840) (powershell-6) |
[reference/6/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/6/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-6&branch=pr-en-us-3840) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C11%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903111446273335-3840%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=b0%2BRMqhKlexPZ2Iorm3EZ5baCwDf1MxBtvPTsD6qIoQ%3D&st=2019-03-11T14%3A55%3A45Z&se=2019-04-11T15%3A00%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/Get-Content.md [hunk 1]
```diff
@@ -50,31 +49,33 @@ If you need to create files or directories for the following examples, see [New-
...
 
-The `Set-Content` cmdlet uses the **Path** and **Value** parameters to create the **Test1.txt** file
-in the current directory. The **Value** parameter includes the text string **Hello, World**. The
-`Get-Content` cmdlet uses the **Path** parameter to specify the **Test1.txt** file and displays the
-content in the PowerShell console.
+### Example 2: Limit the number of lines Get-Conent returns
 
-### Example 2: Get content from a text file and write the content to another file
+This command gets the first five lines of a file. The **TotalCount** parameter is used to gets the
+first 5 lines of content
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Management/Get-Content.md]: 5 should be spelled out on this line.

Comment (bobbytreed) [reply reference/3.0/Microsoft.PowerShell.Management/Get-Content.md]: Fixed

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[f1fa2e7](https://github.com/bobbytreed/PowerShell-Docs/pull/3840/commits/f1fa2e754b0740b716d4c3580491c9b780d5f57f)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/MicrosoftDocs/PowerShell-Docs, working directory: W:\t433-s, target branch: staging, target branch commit: daae16021bba5c886a6e84c1a7df4dd94bb68c1c, source branch: 92f3d678448082a7128abb2c62e5b884/bobbytreed-issue3809, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C12%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903120957054735-3840%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=iGgMRNvO8tSM9EW2OAbv0UD529w2j5q8TpmgS0yBAtQ%3D&st=2019-03-12T09%3A52%3A38Z&se=2019-04-12T09%3A57%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[5d10753](https://github.com/bobbytreed/PowerShell-Docs/pull/3840/commits/5d10753fad9c67bd276e032a73a1a037bfacb46c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/3.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-3.0&branch=pr-en-us-3840) (powershell-3.0) |
[reference/3.0/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/3.0/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-3.0&branch=pr-en-us-3840) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/4.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-4.0&branch=pr-en-us-3840) (powershell-4.0) |
[reference/4.0/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/4.0/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-4.0&branch=pr-en-us-3840) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/5.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.0&branch=pr-en-us-3840) (powershell-5.0) |
[reference/5.0/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/5.0/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-5.0&branch=pr-en-us-3840) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/5.1/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.1&branch=pr-en-us-3840) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/5.1/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-5.1&branch=pr-en-us-3840) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/6/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-6&branch=pr-en-us-3840) (powershell-6) |
[reference/6/Microsoft.PowerShell.Management/Restart-Computer.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3809/reference/6/Microsoft.PowerShell.Management/Restart-Computer.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/restart-computer?view=powershell-6&branch=pr-en-us-3840) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C12%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903121307415428-3840%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=fsdqk0mOD8SUYeYW96SU1iSRuIcLTEiRKdYZfT%2Fy0Js%3D&st=2019-03-12T13%3A07%3A49Z&se=2019-04-12T13%3A12%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**